pvlib/temperature.py

```diff
@@ -16,6 +16,21 @@
     'pvsyst': {'freestanding': {'u_c': 29.0, 'u_v': 0},
                'insulated': {'u_c': 15.0, 'u_v': 0}}
 }
+"""Dictionary of temperature parameters organized by model.
+
+There are keys for each model at the top level. Currently there are two models,
+``'sapm'`` for the Sandia Array Performance Model, and ``'pvsyst'``. Each model
+has a dictionary of configurations; a value is itself a dictionary containing model
```

Comment:
@mikofski can you fix this line length issue? ok to ignore the one in the example (or add # noqa: E501)